recaptcha_enterprise/snippets/src/main/java/recaptcha/accountdefender/SearchRelatedAccountGroupMemberships.java

```diff
@@ -45,7 +45,7 @@ public static void searchRelatedAccountGroupMemberships(
 
       SearchRelatedAccountGroupMembershipsRequest request =
           SearchRelatedAccountGroupMembershipsRequest.newBuilder()
-              .setProject(projectId)
+              .setProject("projects/" + projectId)
```

Comment:
nit: is it necessary? how did it work before this change?

Reply:
This was a recent API change from the product.